doc/.yarnrc.yml

```diff
@@ -1,5 +1,3 @@
 compressionLevel: mixed
 
 enableGlobalCache: false
-
-yarnPath: .yarn/releases/yarn-4.3.1.cjs
```

Comment:
Why do we need to remove this?

Reply:
When we try to run `yarn install` with this line, we get
```
Internal Error: ENOENT: no such file or directory, stat '/Users/daniilankusin/monomer/doc/.yarn/releases/yarn-4.3.1.cjs'
Error: ENOENT: no such file or directory, stat '/Users/daniilankusin/monomer/doc/.yarn/releases/yarn-4.3.1.cjs'
```